### DISCUSSION TITLE
Form isDirty

### DISCUSSION THREAD

**Discussion by muhammedkaraoglu** [inertiajs/inertia#897] [Help (Vue)]
Hi, I need to change some data of my form field when the page is first loaded. isDirty is set true. Is there a way to set this value false when my page first opens?
For example 

`
mounted(){
   this.form.isDirty = false;
}`

this doesn't work.

Sorry for my English, I use google translate.

**Comment by jonagoldman** [comment]
Hard to answer without seeing the code you use to change the form field,
but if you set the value **before** it's passed to `useForm()`, then it's like a `default` value:

```js
const form = useForm({
    name: 'foo',
})
```

 and `isDirty` will be `false`:

![Screenshot from 2021-09-11 01-35-41](https://user-images.githubusercontent.com/1297559/132938969-14ccda8f-74ca-4204-a64d-8f5352aca520.png)

**But** now that field is considered a `default` value so if you reset the form with `this.form.reset()`, the value will return to be the default `foo`.
*And* if you update the value manually or with `this.form.name = null`, now `isDirty` will be `true`.

Not sure what you need bat that should help a bit.
